source/agora/consensus/EnrollmentManager.d

```diff
@@ -453,6 +502,38 @@ public class EnrollmentManager
     }
 }
 
+/// test for serialization and deserialization of a Hash array
+unittest
```

Comment:
This seems to just test serialization? It's not testing EnrollmentManager.

Reply:
No, It's not. I will make the unittest with EnrollmentManager.